lib/eco/treemanager.py

```diff
@@ -2218,7 +2218,8 @@ def lbox_expand_test(self, lbox):
         r2.preparse(outer_root, lbox)
         r2.parse_after(lbox)
         filtered = []
-        for e in r.possible_ends:
+        # XXX filter by distance
```

Comment:
`XXX`?

Reply:
This was a reminder that I need to filter the possible results for expanding a language box by removing all non-maximal results (similar to the creating of language boxes). However, now that I have implemented this I am unsure if that is what we want. Because it means that we will always just expand to the longest match and not show other possible expansion options. Thoughts?

Reply:
We might be able to determine this experimentally: if you run the experiment with both, does the number of acceptable outcomes differ in a meaningful way?

Reply:
I can try. We will likely see less "No insertion (Multi)", since we will always pick the longest match. This won't tell us though if the other options are something the user wanted to do since they may not always want to expand to the longest match.

Reply:
Your call. I don't really know what's best, either.

Reply:
I'm always a bit reluctant removing options from the user, so I think I'm just gonna leave it as is for now and remove this comment. I've saved the filtering code, so we can bring it back if we decide it's the better option.

Reply:
I think we should run the experiment, because it's an almost zero-cost thing to do. If, as is likely, it doesn't tell us very much then we can keep things as-is. Otherwise, AFAICS, we'll never have any other data to guide our decision...

Reply:
Okay, I'll start a run and see what happens.

Reply:
Looking at the results, this change didn't really seem to improve our overall results, so I'm just leaving things as is, as I believe multiple options are in this case better than no options.